src/applications/vaos/tests/appointment-list/components/CommunityCareAppointmentDetailsPage.unit.spec.jsx

```diff
@@ -95,7 +95,7 @@ describe('VAOS <CommunityCareAppointmentDetailsPage>', () => {
     expect(screen.getByRole('link', { name: /7 0 3. 5 5 5. 1 2 6 4./ })).to.be
       .ok;
     expect(
-      screen.getByRole('heading', { level: 2, name: /Special instructions/ }),
+      screen.getByRole('heading', { level: 4, name: /Special instructions/ }),
```

Comment:
I don't think this should have changed. You might need to check the CommunityCareAppointmentDetailsPage and see if it's using the AppointmentInstructions component for the current homepage and update it if so.

Reply:
Fixed and adjusted the instructions logic 
![image](https://user-images.githubusercontent.com/8315447/110011662-b0309780-7ced-11eb-9fa5-73a0c38809e1.png)